server/game/cardImplementations/02_SHD/GreefKargaAffableCommissioner.ts

```diff
@@ -15,7 +15,7 @@ export default class GreefKargaAffableCommissioner extends NonLeaderUnitCard {
             title: 'Search the top 5 cards of your deck for an upgrade, then reveal and draw it.',
             immediateEffect: AbilityHelper.immediateEffects.deckSearch({
                 searchCount: 5,
-                cardCondition: (card) => card.type === CardType.Upgrade,
+                cardCondition: (card) => card.isUpgrade(),
```

Comment:
The two are exactly equivalent in behavior. What you're suggesting would be to filter on something like `CardType.BasicUpgrade`, which is doable. But as you observe, there will never be tokens in the deck, so it would just be for additional clarity for the reader.

I'm fine either way, just lmk your preference.